src/runtime/composables/useDirectusToken.ts

```diff
@@ -1,14 +1,86 @@
 import { useCookie, useNuxtApp, useRuntimeConfig } from "#app";
+import { CookieRef } from "~~/node_modules/nuxt3/dist/app/composables/cookie";
+import { DirectusAuthResponse } from "../types";
+
 
 export const useDirectusToken = () => {
   const nuxtApp = useNuxtApp();
+  const baseURL = useDirectusUrl();
+  const config = useRuntimeConfig()
+
+  const accessToken = (): CookieRef<string> => {
```

Comment:
Dont import from `node_modules`, please find another way for `CookieRef`

Reply:
Maybe I can remove the type annotation. No need for it here. Type is clear.

Reply:
That would be great!